style.css

```diff
@@ -0,0 +1,159 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Open Sans", sans-serif;
+}
+/* 
+  style the Header section 
+  */
+header {
+  background-color: #f8f8f8;
+  border-color: #e7e7e7;
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  height: 40px;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  border-style: solid;
+  border-width: thin;
+  padding: 0px 15px;
+}
+nav {
+  width: 100%;
+}
+
+/* Style the links inside the header section */
+ul li a {
+  text-decoration: none;
+  cursor: pointer;
+  color: #777777;
+}
+
+.logo {
+  width: 265px;
+  height: 50px;
+  cursor: pointer;
+}
+
+/*  style the elemints inside the navegation bar */
+#benefit-item,
+#blog-item,
+#login-item,
+#resister-item,
+#work-item {
+  height: 40px;
+  font-size: 13px;
+  border-style: solid;
+  border-width: thin;
+  border-color: #d3d3d3;
+  border-top: none;
+  text-align: center;
+  font-weight: 400px;
+  display: block;
+}
```

Comment:
you can add to all of these items the same class name and apply this style to it.